lib/plausible/stats/query_result.ex

```diff
@@ -70,7 +70,10 @@ defmodule Plausible.Stats.QueryResult do
 
   defp meta(query) do
     %{
-      warning:
+      imports_included: if(query.include.imports, do: query.include_imported, else: nil),
```

Comment:
I'm a little confused by this line here. I think it should always be a boolean. Wouldn't it work simply like this?

```suggestion
      imports_included: query.include.imports,
```

Reply:
Oops. This doesn't work and the tests luckily caught that. Intent is to signal whether imports were included successfully with this flag.